pkg/controller/atlasfederatedauth/atlasfederated_auth_controller_test.go

```diff
@@ -0,0 +1,216 @@
+package atlasfederatedauth
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"go.mongodb.org/atlas-sdk/v20231115004/admin"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	atlas_mock "github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/mocks/atlas"
```

Comment:
https://go.dev/blog/package-names

I would avoid underscores:

```suggestion
	atlasmock "github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/mocks/atlas"
```

Reply:
agreed, fixed 👍